openy_editor/config/install/editor.editor.full_html.yml

```diff
@@ -62,8 +62,6 @@ settings:
         - rel
         - target
         - title
-    media_media:
-      allow_view_mode_override: false
```

Comment:
@david-storm I think this needs to be `allow_view_mode_override: false` if we want to allow the override.